pset-iterators.js

```diff
@@ -184,6 +231,18 @@ console.log('Problem 7:')
 
 // Add your code below this line
 
+const shortTermPresidents = presidents.filter((president) => {
+	const tookOffice = parseInt(president.took_office)
+	const leftOffice = parseInt(president.left_office)
+	if(leftOffice - tookOffice < 4) {
+		return president.president
+	}
+}).map((president) => {
+    return president.president
+})
+
+console.log(`The list of short term presidents is as follows: ${shortTermPresidents}`)
+
```

Comment:
@cswebdesign Good job here on this solution. The filter method was a good choice to use here and using the parseInt on `president.took_office` and `president.left_office` to get back the year value in terms of numbers.
For this example, you didn't need a chained map method because you are returning an array of objects.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L201)